system/ee/ExpressionEngine/Addons/structure/views/index.php

```diff
@@ -145,10 +145,18 @@
                     }
                 }
 
-                if ($page['hidden'] == 'y') {
+                if (isset($settings['show_entry_status']) && $settings['show_entry_status'] == 'y') {
+                    echo isset($status_tags[$page['status']]) ? $status_tags[$page['status']] : '<span class="status-tag missing-status">' . $page['status'] . '</span>';
+                }
+
+                if ((!isset($settings['show_nav_status']) || $settings['show_nav_status'] == 'y') && $page['hidden'] == 'y') {
                     echo ' <span class="hidden-page">(hidden)</span>';
                 }
 
+                if (isset($page['extra_html']) && $page['extra_html']) {
+                    echo ' <span class="extra-html">' . $page['extra_html'] . '</span>';
```

Comment:
Added this as a way to use the `structure_index_view_data` hook to add even further UI elements as needed. For instance, additional Publisher tags:

<img width="416" alt="image" src="https://github.com/ExpressionEngine/ExpressionEngine/assets/2423727/6e8b5077-06fc-409e-a1b3-12c3a53f56b8">